packages/app-store/_utils/oauth/OAuthManager.ts

```diff
@@ -409,10 +458,9 @@ export class OAuthManager {
   }
 
   // TODO: On regenerating access_token successfully, we should call makeTokenObjectValid(to counter invalidateTokenObject). This should fix stale banner in UI to reconnect when the connection is working
-  private async refreshOAuthToken() {
+  private async refreshOAuthToken({ refreshToken }: { refreshToken: string | null }) {
```

Comment:
Pass refresh token directly from the place we are sure `currentTokenObject` is available. Keeps TS happy